src/ariel-os-sensors/src/sample.rs

```diff
@@ -96,6 +122,9 @@ pub enum Accuracy {
         /// [`bias`](Accuracy::SymmetricalError::bias).
         scaling: i8,
     },
+    /// Indicates that the value from this channel is temporarily unavailable and that the [`Sample`]'s value is considered opaque and should be ignored.
```

Comment:
I think we can remove the end of that sentence as we're removing `raw_value()` and the raw value is therefore not exposed anymore in this case:
```suggestion
    /// Indicates that the value from this channel is temporarily unavailable.
```